cmd/Dockerfile

```diff
@@ -1,7 +1,25 @@
+FROM quay.io/centos/centos:stream8 as builder
+
+RUN mkdir /workdir
+WORKDIR /workdir
+
+COPY go.mod .
+RUN dnf install -y golang-$(sed -En 's/^go +(.*+)$/\1/p' go.mod).*
+
+COPY . .
```

Comment:
we can just use the normal golang image and pass the tag with the version outside of this so we don't have to install

Reply:
Issue is that the only legit pubilc golang builder I was able to find is on Docker Hub. And using that would mean facing the pull limiting...